Source/Collections/INullFreeDictionary.cs

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I don't think an interface here adds anything.

Reply:
Yea, not really